Index.html

```diff
@@ -75,14 +75,61 @@
                         <input type="text" placeholder="Search GitHub">
                     </li>
                     <li><a href="#">Sign in</a></li>
-                    <li><button class="button"><a href="#">Sign up</a></button></li>
+                    <li><button class="button1"><a href="#">Sign up</a></button></li>
                 </ul>
                 <!-- #endregion -->
             </nav>
-
         </div>
     </header>
 
+    <section class="section-1">
+        <div class="text-div1">
+        <h1>Built for developers</h1><br>
+        <p class="grey-text">GitHub is a development platform inspired by the way you work. From <span class="underline"><a><u>open source</u></a></span> to <span class="underline"<a><u>business</u></a></span>, you can host and review code, manage projects, and build software alongside 50 million developers.</p>
+        </div>
+        <div class="form-div">
+            <form>
+                <label class="form-label" for="Username">Username</label><br>
+                <input class="form" type="text" id="username" name="username"><br>
+                <label for="email">Email</label><br>
+                <input class="form" type="text" id="email" name="email"><br>
+                <label for="password">Password</label><br>
+                <input class="form" type="text" id="password" name="password"><br>
+            </form>
+            <br>
+            <p style="font-size: 11px;">Make sure it's at least 15 characters OR at least 8 characters including a number and a lowercase letter. <a>Learn more.</a>
+            </p>
+            <br>
+            <button class="green-button">Sign up for GitHub</button>
+            <br><br><br><br>
+            <p style="font-size: 11px;">By clicking “Sign up for GitHub”, you agree to our Terms of Service and Privacy Statement. We’ll occasionally send you account related emails.</p>
+        </div>
+    </section>
+
+    <section class="section2">
+        <img class="small-image" src="assets/images/overlay.svg" alt="overlay svg">
+        <div class="div-box">
+            <div class="text-div2">
+                <h2>Get started with GitHub Enterprise</h2><br>
+                <p>Take collaboration to the next level with security and administrative features built for teams.</p>
+            </div>
+            <div class="text-div3">
+                    <img src="assets/images/enterprise.svg" alt="Enterprise SVG"><br>
+                    <br><p><strong>Enterprise</strong></p><br>
+                    <p>Deploy to your environment or the cloud.</p><br><br>
+                    <button class="blue-button">Start a free trial</button>
+            </div>
+            <div class="text-div4">
```

Comment:
Please can you go through and just replace the numbers on the classes for all, I won't add any more comments for this then